end-to-end-tests/specs/results.spec.js

```diff
@@ -24,7 +21,7 @@ describe('Results Page', function() {
         describe('single study query with four genes', ()=>{
             before(()=>{
                 var url = `${CBIOPORTAL_URL}/index.do?tab_index=tab_visualize&cancer_study_list=coadread_tcga_pub&cancer_study_id=coadread_tcga_pub&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=coadread_tcga_pub_mutations&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=coadread_tcga_pub_gistic&Z_SCORE_THRESHOLD=2.0&case_set_id=coadread_tcga_pub_nonhypermut&case_ids=&gene_list=BRAF+KRAS+NRAS&gene_set_choice=user-defined-list&Action=Submit#pancancer_study_summary`;
-                browser.url(url);
+                goToUrlAndSetLocalStorage(url);
```

Comment:
removed all references to `browser.url`, always use `goToUrlAndSetLocalStorage`. There's no extra cost to this now, because we no longer do a refresh